ruby-focal/3.0.x/Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM socrata/base-focal
+LABEL maintainer="Socrata 'sysadmin@socrata.com'"
```

Comment:
I also changed this from the deprecated MAINTAINER to a LABEL, using the format here: https://github.com/socrata/shipyard/blob/main/base-focal/Dockerfile#L2